### PR TITLE
Fix last entity for new search procedure

### PR DESCRIPTION
**Pull request by oshtaier**
Fixing container class for new search procedure. Also fixing general search procedure for long strings. Add description for custom search methods

### CODE REVIEW

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/ui/test_docker.py
..SS...SS...SSS.......................E....S
----------------------------------------------------------------------
======================================================================
ERROR: @Test: Create Docker-type repositories on multiple products and
======================================================================
Ran 44 tests in 4761.086s

FAILED (SKIP=8, errors=1)
```
```
nosetests tests/foreman/ui/test_docker.py -m test_delete_random_docker_repo
.
----------------------------------------------------------------------
Ran 1 test in 151.066s

OK
```

File: robottelo/constants.py [hunk 1]
```diff
@@ -728,3 +728,12 @@
               'Certified Cloud Providers (RH CCP)'),
     'usgcb': 'United States Government Configuration Baseline (USGCB)',
 }
+
+SEARCH_EXCEPTIONS_LIST = [
+    'Location',
+    'Organization',
```
Comment (kbidarkar) [inline robottelo/constants.py]: why "organization" is part of the exception list , it is katello entity ?

Comment (oshtaier) [inline robottelo/constants.py]: Maybe yes, but `is_katello = False` for it as it has old style for search bar and search button

**Comment by kbidarkar** [comment]
ACK

**Comment by abalakh** [comment]
Test result for smoke test:
```
nosetests tests/foreman/smoke/test_ui_smoke.py -m test_smoke        
.
----------------------------------------------------------------------
Ran 1 test in 404.484s

OK
```

**Comment by abalakh** [comment]
ACK

**Pull request merged**